hawc/apps/lit/forms.py

```diff
@@ -548,3 +549,50 @@ def clean_excel_file(self):
                 '"Full text URL", case sensitive.'
             )
         return fn
+
+
+class BulkReferenceStudyExtractForm(forms.Form):
+    reference_ids = forms.ModelMultipleChoiceField(
+        widget=forms.CheckboxSelectMultiple, queryset=models.Reference.objects.none(), required=True
+    )
+    study_type = forms.TypedMultipleChoiceField(
+        widget=forms.CheckboxSelectMultiple,
+        choices=[(1, "Bioassay"), (2, "Epidemiology"), (3, "Epi Meta"), (4, "In Vitro")],
```

Comment:
instead of ints, can we just use the values that are on the model? For example, 1 could be "bioassay", 2 could be "epi", etc. That way you don't need that custom build-up in the bulk_create_studies.  

You can do 

```python
for st in study_type:
    attrs.update(st=True)
```